packages/react-ui-core/src/Form/Input.js

```diff
@@ -70,6 +70,11 @@ export default class Input extends Component {
      * Used to determine if InputMask required
      */
     mask: PropTypes.string,
+
+    /**
+     * Adds ability to pass ref down to the actual DOM node
+     */
+    componentRef: PropTypes.func,
```

Comment:
I think you want to use this: https://reactjs.org/docs/forwarding-refs.html#forwarding-refs-in-higher-order-components

Reply:
This only sends the ref down one level. We need it to go down about 6 levels or so, so I figured this was cleaner.